template/config/webpack.config.prod.js

```diff
@@ -92,7 +92,7 @@ webpackConfig.plugins = [...webpackConfig.plugins,
     }
   }),
   new UglifyJsPlugin({
-    include: /\.min\.js$/,
+    include: /\.js$/,
```

Comment:
Thanks for this - I've been meaning to do it for a while :)

Reply:
yes, you already merged my other PR: https://github.com/ducksoupdev/vue-webpack-typescript/pull/26/files